### ISSUE TITLE
Unwanted debug traces in PMP refining

### ISSUE THREAD

**Issue by seb-tourneux** [CGAL/cgal#2600]
Debug traces are displayed by default on modules using `Polygon_mesh_processing/internal/refine_impl.h`

In two places :
Polygon_mesh_processing/internal/refine_impl.h:173
```
    #ifndef CGAL_PMP_REFINE_DEBUG
    std::cerr << "Test " << interior_edges.size() << " edges " << std::endl;
    #endif
```
Polygon_mesh_processing/internal/refine_impl.h:184
```
    #ifndef CGAL_PMP_REFINE_DEBUG
    std::cerr << "Test " << interior_edges.size() << " edges " << std::endl;
    #endif
```

I guess the #ifndef should be replaced by #ifdef, as for instance `Polygon_mesh_processing/internal/refine_impl.h:326`

This can be tested executing `Polygon_mesh_processing::refine`

Thanks

**Comment by lrineau** [comment]
Added by @sloriot in a3666887a1c. -> CGAL-4.10.

**Comment by lrineau** [comment]
Fixed by #2602.
